Gems/Atom/RHI/Code/Include/Atom/RHI/CommandList.h

```diff
@@ -85,11 +86,42 @@ namespace AZ
             /// End predication on the command list.
             virtual void EndPredication() = 0;
 
-            //! Builds a Bottom Level Acceleration Structure (BLAS) for ray tracing operations, which is made up of RayTracingGeometry entries
+            /// Builds a Bottom Level Acceleration Structure (BLAS) for ray tracing operations, which is made up of RayTracingGeometry entries
             virtual void BuildBottomLevelAccelerationStructure(const RHI::RayTracingBlas& rayTracingBlas) = 0;
 
-            //! Builds a Top Level Acceleration Structure (TLAS) for ray tracing operations, which is made up of RayTracingInstance entries that refer to a BLAS entry
+            /// Builds a Top Level Acceleration Structure (TLAS) for ray tracing operations, which is made up of RayTracingInstance entries that refer to a BLAS entry
             virtual void BuildTopLevelAccelerationStructure(const RHI::RayTracingTlas& rayTracingTlas) = 0;
+
+            /// Defines the submit range for a CommandList
+            /// Note: the default is 0 items, which disables validation for items submitted outside of the framegraph
+            struct SubmitRange
+            {
+                // the zero-based start index of the range
+                uint32_t m_startIndex = 0;
+
+                // the end index of the range
+                // Note: this is an exclusive index, meaning submitted item indices should be less than this index
+                uint32_t m_endIndex = 0;
+
+                // returns the number of items in the range
+                uint32_t GetCount() const { return m_endIndex - m_startIndex; }
```

Comment:
add 'inline' keyword

Reply:
The inline keyword is not necessary if the method implementation is in the class definition, it is automatically inlined.